src/components/AutoCompleteInput/AutoCompleteInput.js

```diff
@@ -0,0 +1,144 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import styled, { css } from 'react-emotion';
+import Downshift from 'downshift';
+
+import SearchInput from '../SearchInput';
+import Card from '../Card';
+import { textMega } from '../../styles/style-helpers';
+
+const AutoCompleteWrapper = styled('div')`
+  label: input__container
+  position: relative;
+  min-width: 150px;
+`;
+
+const ItemsWrapper = styled('div')`
+  position: relative;
+  height: 0px;
+  overflow: visible;
+  margin-top: ${props => props.theme.spacings.bit};
+`;
+
+const itemsBaseStyles = ({ theme }) => css`
+  padding-top: calc(${theme.spacings.mega} - ${theme.spacings.bit});
+  padding-bottom: calc(${theme.spacings.mega} - ${theme.spacings.bit});
+  min-width: initial !important;
+  position: absolute;
+  top: 0;
+  left: 0;
+  right: 0;
+`;
+
+const Items = styled(Card)(itemsBaseStyles);
+
+Items.defaultProps = Card.defaultProps;
+
+const itemBaseStyles = ({ theme }) => css`
+  cursor: pointer;
+  padding: ${theme.spacings.bit} 0 ${theme.spacings.bit} 0;
+  margin: 0;
+  text-overflow: ellipsis;
+  overflow: hidden;
+  ${textMega({ theme })};
+`;
+
+const itemHighlight = ({ selected, theme }) =>
+  selected &&
+  css`
+    color: ${theme.colors.b500};
+  `;
+
+const Item = styled('div')(itemBaseStyles, itemHighlight);
+
+const filterItems = inputValue => item =>
+  !inputValue || item.toLowerCase().includes(inputValue.toLowerCase());
```

Comment:
Also here, no `.includes`. Sorry! We need to document this or add some linting rules.